skimage/measure/_find_contours.py

```diff
@@ -9,7 +9,7 @@
 
 
 @deprecate_kwarg({'array': 'image'}, removed_version="0.20")
-def find_contours(image, level,
+def find_contours(image, level='median',
```

Comment:
Setting the default to a string may suggest that other string values are accepted ('mean' for example)
```suggestion
def find_contours(image, level=None,
```